graylog2-web-interface/src/components/bootstrap/BootstrapModalWrapper.jsx

```diff
@@ -53,6 +65,7 @@ BootstrapModalWrapper.propTypes = {
   ]).isRequired,
   onOpen: PropTypes.func,
   onClose: PropTypes.func,
+  onCancel: PropTypes.func,
```

Comment:
I don't see why we need this prop here, `BootstrapModalWrapper` has nothing to do with cancelling, or forms. It is simply a wrapper to make handling modals a bit easier, or at least that is its motivation.